app/src/main/java/com/duckduckgo/app/bookmarks/ui/EditSavedSiteDialogFragment.kt

```diff
@@ -20,15 +20,18 @@ import android.os.Bundle
 import android.text.Editable
 import android.view.View
 import android.widget.EditText
+import com.duckduckgo.app.bookmarks.model.BookmarkFolder
 import com.duckduckgo.app.bookmarks.model.SavedSite
 import com.duckduckgo.app.bookmarks.ui.bookmarkfolders.AddBookmarkFolderDialogFragment
+import com.duckduckgo.app.bookmarks.ui.bookmarkfolders.BookmarkFoldersActivity
 import com.duckduckgo.app.browser.R
 import com.duckduckgo.app.global.view.TextChangedWatcher
 
 class EditSavedSiteDialogFragment : SavedSiteDialogFragment() {
 
     interface EditSavedSiteListener {
-        fun onSavedSiteEdited(savedSite: SavedSite)
+        fun onSavedSiteFavoriteEdited(favorite: SavedSite.Favorite)
```

Comment:
I would try to avoid creating 2 methods here if it's not really necessary.

Checking the implementations of `onSavedSiteBookmarkEdited` only 1 of the 3 is using bookmarkFolder -> `BrowserTabViewModel`. I will leave a comment there.

Reply:
Yes removing the bookmark folder from the parameter helped me remove this bad code.